Jenkinsfile

```diff
@@ -60,7 +60,7 @@ node('ca-jenkins-agent') {
     ]
 
     // Initialize the pipeline library, should create 5 steps
-    pipeline.setup()
+    pipeline.setup(nodeJsVersion: 'v12.22.3')
```

Comment:
Curious if we should be updating the node version pretty soon to 14+ ?

Reply:
Have been using v12 in Jenkinsfiles across all the Zowe CLI components, but this is a good point since Node 12 reaches EoL in Apr 2022. I suppose it's fine to keep using it for now - when v12 loses support, we'll have to update the version matrix in our GH workflows and could update the version here too. Or maybe by then we'll have migrated away from Jenkins 🙂